handlers/grpcpanichandler.go

```diff
@@ -0,0 +1,29 @@
+package handlers
```

Comment:
should be middleware package, handlers usually at service level

Reply:
changed to middleware.